.github/workflows/publish.yml

```diff
@@ -0,0 +1,50 @@
+# This workflow will upload a Python Package using Twine when a release is created
+# For more information see: https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#publishing-to-package-registries
+
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+name: Upload Python Package
+
+on: push
+
+jobs:
+  release:
+    name: Create Github release
+    runs-on: ubuntu-20.04
+    if: startsWith(github.event.ref, 'refs/tags/v') && endsWith(github.event.ref, '.0')
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Release
+        uses: softprops/action-gh-release@v1
+        with:
+          body_path: RELEASE.md
```

Comment:
This file does not yet exist, right?
That would make the CI fail

Reply:
Right, I just created the file.

Reply:
But we should also fill the file with some content, otherwise we end up with empty release messages.

You can recycle most of the README into the Release file. One sentance about the package, the pip install instructions and how to call the help page of CLI , for example

Reply:
Ah okay, I thought the file is automatically filled with the release notes.
I just added some information about metacatalog to `RELEASE.md`.